app/src/main/java/mono/hg/LauncherActivity.java

```diff
@@ -938,7 +938,8 @@ private void addPanelListener() {
 
             @Override public void onPanelStateChanged(View panel, int previousState, int newState) {
                 if (newState == SlidingUpPanelLayout.PanelState.COLLAPSED
-                        || newState == SlidingUpPanelLayout.PanelState.DRAGGING) {
+                        /*|| newState == SlidingUpPanelLayout.PanelState.DRAGGING*/) {
```

Comment:
This does work, but causes the search bar to appear late. We can probably add a boolean somewhere to better its use.

Reply:
ok. i will try some other ways today.